Dockerfile

```diff
@@ -35,3 +35,20 @@ COPY --from=build /go/bin/velero-plugin-for-gcp /plugins/
 COPY --from=build /go/bin/cp-plugin /bin/cp-plugin
 USER 65532:65532
 ENTRYPOINT ["cp-plugin", "/plugins/velero-plugin-for-gcp", "/target/velero-plugin-for-gcp"]
+
+ARG CREATED
+ARG VERSION
+ARG GIT_SHA
+
+LABEL \
+	org.opencontainers.image.created=${CREATED} \
+	org.opencontainers.image.url="https://hub.docker.com/r/velero/velero-plugin-for-gcp" \
+	org.opencontainers.image.documentation="https://velero.io" \
```

Comment:
Actually, for this image specifically, doc should be `https://github.com/vmware-tanzu/velero-plugin-for-gcp` imo.